fenix/app/.experimenter.yaml

```diff
@@ -139,25 +139,31 @@ re-engagement-notification:
     type:
       type: int
       description: The type of re-engagement notification that is shown to the inactive user.
-search-term-groups:
-  description: A feature allowing the grouping of URLs around the search term that it came from.
+search-extra-params:
+  description: A feature that provides additional args for search.
   hasExposure: true
   exposureDescription: ""
   variables:
+    channel-id:
+      type: json
+      description: The channel Id param name with arg.
     enabled:
       type: boolean
-      description: "If true, the feature shows up on the homescreen and on the new tab screen."
-search_extra_params:
-  description: A feature that provides a search engine name and a channel ID.
+      description: "If true, the feature is active."
+    feature-enabler:
+      type: json
+      description: "The feature enabler param name with arg, NOTE this map could be empty."
```

Comment:
Rather than having seaprate channel-id and feature-enabler lists, would it not be simpler to have something like:

```
parameters:
  type: json
  description: "One or more name/value pairs to be appended to the URL for the engine"
```

Reply:
It can be done, sure. Although in this case, the key params, `tpfs` and `channel` need to be hardcoded or the `tpfs`/`channel` key value pair needs to be defined as the first (or second) entry of the map.

As of this implementation, nothing is hardcoded and can be customized fully.